### ISSUE TITLE
Tab preview animation on Windows 7 and Windows 10

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#2869]
**Describe the issue you encountered:** Tab preview animation seems to be not working properly on Windows 7. It shows the preview on hover. Tested on VM.

![tabpreview](https://cloud.githubusercontent.com/assets/3362943/17328660/dca9970c-58f9-11e6-82e6-607ae5174d5a.gif)

**Expected behavior:**
- Platform (Win7, 8, 10? macOS? Linux distro?): Windows 7
- Brave Version: 0.11.2 Beta2
- Steps to reproduce:
  1.
  2.
  3.
- Screenshot if needed:
- Any related issues: #2862

**Comment by srirambv** [comment]
Tested on Desktop running Windows 7 Prof. There is a very little blur effect in tab preview but not as prominent as in VM.

**Comment by luixxiul** [comment]
This can be reproduced on Win10 VM. Maybe this should be labelled ``wontfix``.

**Comment by cezaraugusto** [comment]
Confused here. Did you tested after #2817 being merged? What should be the expected result?

**Comment by luixxiul** [comment]
@cezaraugusto I tested it on 0.11.2 Beta2 and think it has been merged. The problem I think is that when I hover the tab the preview is displayed at that moment, and then the tab starts brightening and darkening.

**Comment by cezaraugusto** [comment]
I had to change transitions to fix the flickering tab issue, maybe it's what you're seeing. Needs a little polish tho. I'll check with Brad how can we improve it.

**Comment by bradleyrichter** [comment]
@cezaraugusto your last video looked much smoother. Strange.

Basically, the tabs are not cross-fading. At one point you had them crossfading with the active tab showing in the middle of the sequence. 

I was not able to preview any of your changes after that feedback.


example Tabs                 Tab 1    -         Tab 2    -          Tab 3    -         Tab 4
state                               Active             Preview           Preview            Preview
transition                        none            ease-in/out      ease-in/out      ease-in/out


There may be a problem in the structure that prevents the above approach?

**Comment by cezaraugusto** [comment]
@bradleyrichter it's like we talked on chat. :)

Assigning it to myself until I polish tab transitions.

**Comment by srirambv** [comment]
Verified on Windows x64 
* 0.22.6  e6ff4ea
* libchromiumcontent: 65.0.3325.162
* muon: 5.1.0
